crates/paimon/src/io/file_io.rs

```diff
@@ -155,6 +165,9 @@ impl FileIO {
 #[derive(Clone, Debug)]
 pub struct FileStatus {
     pub size: u64,
+    pub is_dir: bool,
+    pub path: String,
+    pub last_modification_time: i64,
```

Comment:
I'm thinking about using [`chrono::DateTime`](https://docs.rs/chrono/latest/chrono/struct.DateTime.html) to make it easier to use. What do you think?

Reply:
Make sense, updated.